src/app/filter/filter-results.component.ts

```diff
@@ -92,7 +92,7 @@ export class FilterResultsComponent implements DoCheck, OnInit {
     let newFilters: Filter[] = [];
     this.config.appliedFilters.forEach((appliedFilter) => {
       if (appliedFilter.field.title !== filter.field.title
-          || appliedFilter.value !== filter.value) {
+        || appliedFilter.value !== filter.value) {
         newFilters.push(appliedFilter);
```

Comment:
Don't like how the indentation is removed here. I've always indented this way (a Java standard) for readability.